deploy-manage/deploy/cloud-enterprise/deploy-medium-installation.md

```diff
@@ -1,9 +1,13 @@
 ---
+applies_to:
+  deployment:
+    ece: all
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-install-medium-cloud.html
+  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-install-medium-onprem.html
 ---
 
-# Deploy a medium installation cloud [ece-install-medium-cloud]
+# Deploy a medium installation [ece-install-medium]
 
 This type of installation is recommended for many production setups. You need:
```

Comment:
at the bottom, let's make this 3 different code snippets that are labeled (easier to copy)